src/query/list_file.c

```diff
@@ -5050,32 +5085,49 @@ qfile_finalize_list_cache (THREAD_ENTRY * thread_p)
  *   list_ht_no(in)     :
  */
 int
-qfile_clear_list_cache (THREAD_ENTRY * thread_p, int list_ht_no)
+qfile_clear_list_cache (THREAD_ENTRY * thread_p, XASL_CACHE_ENTRY * xcache_entry, bool invalidate)
 {
   int rc;
-  bool del = true;
   int cnt;
+  int list_ht_no;
 
-  if (QFILE_IS_LIST_CACHE_DISABLED)
+  if (csect_enter (thread_p, CSECT_QPROC_LIST_CACHE, INF_WAIT) != NO_ERROR)
     {
-      return ER_FAILED;
+      goto end;
+      //return ER_FAILED;
```

Comment:
Is it the redundant comment? or Is error handling necessary here?

Reply:
Ok, I'll remove it.